src/components/standup/FormInputComponent.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+
+import styles from '@/components/standup/standupContainer.module.scss';
+import { InputProps } from '@/interfaces/standup.type';
+
+const FormInputComponent: FC<InputProps> = ({
+    placeholder,
+    name,
+    value,
+    dataTestId,
+    handleChange,
+}) => {
+    return (
+        <input
+            data-testid={dataTestId}
+            type="text"
+            className={styles.inputField}
+            placeholder={placeholder}
+            required
+            name={name}
+            value={value}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                handleChange(e)
+            }
+        />
+    );
+};
+
+export default React.memo(FormInputComponent);
```

Comment:
no point of putting this React.memo as its coming as a prop so your component will keep rendering as many times as parent rerenders

Reply:
fixed